### DISCUSSION TITLE
My friend's account was suspended stating violation of Terms and Condition and now no response toappeal mail

### DISCUSSION THREAD

**Discussion by techy4shri** [community/community#135681] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

So my friend's account was suspended stating "violation of Terms adn Conditions" but he has simply been contributing to opensource projects for the past three months.
After a bit of diggin we got to know that someone from the organisation might have reported his account as he was in top 20 on the leaderboard. While he did fill the contact form to appeal the suspension, there has been no reply, how does he proceed further to get his account back? Four years of both academic and professional code and data lost because of this.

**Comment by ansh3108** [comment]
what's his username?

**Comment by ebndev** [comment]
Hi @techy4shri,

This community is managed by GitHub employees, but it is not a support forum. Your friend done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
